### DISCUSSION TITLE
Is it possible to detect duplicated strings across multiple files?

### DISCUSSION THREAD

**Discussion by Songyu-Wang** [eslint/eslint#14996] [Q&A]
Hello,
I have a scenario that I am trying to detect, but Idk if it is possible.

```javascript
// File 1
foo(){
  doSomethingWithTheString('ABC')
  doSomethingWithTheString('CDE')
}

// File 2
bar(){
  doSomethingWithTheString('ABC')
}

/** lint rule result
*'ABC' was passed into `doSomethingWithTheString` in BOTH File 1 and File2, it is dangerous for reason XXX! 
*/
```

I admit I don't usually work with lint rule that often, but to the best of my knowledge, a lint rule’s scope does not go beyond file level. Just wanna double check with the community and being hopeful.
Thanks!

**Comment by nzakas** [comment]
ESLint only works on one file at a time, so it’s not possible to aggregate warnings like this.
